language/move-binary-format/src/normalized.rs

```diff
@@ -420,3 +706,46 @@ impl std::fmt::Display for Type {
         }
     }
 }
+
+fn struct_name(m: &CompiledModule, idx: &StructDefinitionIndex) -> Identifier {
```

Comment:
This is quite a common sequence of operations, I feel like we should add this to `ModuleAccess` or `BinaryIndexedView` or something like that (if it's not already there).